semesterly/dev_credentials.py

```diff
@@ -25,8 +25,8 @@
     'GOOGLE_API_KEY': 'AIzaSyB4O0Z89wwSWdVFufZMmW7n05ftUaau9cs',
     'SOCIAL_AUTH_GOOGLE_OAUTH2_KEY': '207925266161-qa90rek8apc0ablkjr3565um4u9kiht1.apps.googleusercontent.com',
     'SOCIAL_AUTH_GOOGLE_OAUTH2_SECRET': 'OiVIoVFTZrWXZ87tx7r1KJoo',
-    'SOCIAL_AUTH_FACEBOOK_KEY': '2008148216081463',
-    'SOCIAL_AUTH_FACEBOOK_SECRET': '75f855457ff18db012e57ccaaf3470eb',
+    'SOCIAL_AUTH_FACEBOOK_KEY': '580022102164877',
```

Comment:
You shouldn't be changing this line, correct? 
this is your own private test account

Reply:
Yep, I'll remove it. Should I add this file to .gitignore?